salt/salt.spec

```diff
@@ -1213,29 +1228,16 @@ for script in $DEF_PYPATH/*; do
 done
 
 ## create missing directories
-install -Dd -m 0750 %{buildroot}%{_sysconfdir}/salt/master.d
-install -Dd -m 0750 %{buildroot}%{_sysconfdir}/salt/minion.d
-install -Dd -m 0750 %{buildroot}%{_sysconfdir}/salt/cloud.maps.d
-install -Dd -m 0750 %{buildroot}%{_sysconfdir}/salt/cloud.profiles.d
-install -Dd -m 0750 %{buildroot}%{_sysconfdir}/salt/cloud.providers.d
```

Comment:
@aplanas is it right to remove these lines from the spec file? I don't see them added back afterwards.

Reply:
Are duplicated, you can see for example master.d in 1243. To avoid duplicated I sorted them, as this will help next time to detect duplicates.

There is a separate commit only for this change, can be more easy to inspect.

Reply:
Thanks for clarifying :+1: